### ISSUE TITLE
Problem setting new value for group of checkboxes - model does not update

### ISSUE THREAD

**Issue by bizcmd** [wearebraid/vue-formulate#383]
**Describe the bug**
v2.5.0:  Using v-model on a `FormulateInput` of type `group` with child FormulateInputs of type `checkbox`, setting the model variable to a new object/array does not update the checkbox visuals, and the model is overwritten with member items as empty strings `""`

### CodePen:
CodePen here: https://codepen.io/flyfoxuk/pen/oNYwOBj

**Reproduction**
In the CodePen, click the "Set New Values" button.  Observe the red debug area which outputs the current value of the `groupModel` variable.  It does not update.  The checkbox states do not update either.

**Expected behavior**
The `groupModel` variable should update as set, and retain those settings, and for the visual form to update the checkbox states with the new values.

**Screenshots**
Initial state:
![image](https://user-images.githubusercontent.com/75837683/108489941-12bf7780-729a-11eb-837f-4d6c009c9a49.png)
This is the same state when the Set New Values button is pressed, which does `this.$set` on the `groupModel` variable to attempt to update it:
```
      this.$set(this, 'groupModel', [
        {
          course_A: ['ABC', 'DEF'],
          course_B: ['XYZ']
        }
      ])
```

**Additional Info**
Manually clicking the checkboxes updates the `groupModel` variable as expected, but programmatically setting the `groupModel` to a new array value does not.

Is there a more correct way of trying to update the groupModel variable?

**Device information:**
Mac OS 10.15.5, Safari 13.1.1, Chrome 88.0.4324.96

**Comment by justin-schroeder** [comment]
Thanks for the quality bug report. I can reproduce it. Interesting that it only applied to box elements...I'll tag it for further investigation.

**Comment by wkmpl** [comment]
Hi, is there any fix, workaround already ? Thanks, for your effort.

**Comment by wkmpl** [comment]
in src/libs/context.js on line 534
`  if (this.type === 'checkbox' && !Array.isArray(this[model]) && this.options) {
    return []
  }`

`this[model]` seems to be initially an empty string, don´t know where it comes from and so it will return an empty array here instead of the value set ?

**Comment by justin-schroeder** [comment]
I believe this is fixed in the `release/2.51` branch now 👍
